skins/cat17/templates/partials/donation_confirmation/membership_advantages.html.twig

```diff
@@ -1,12 +1,21 @@
 <div class="container hidden-print">
 	<div class="row">
-		<div class="content col-xs-12 col-md-8 content col-xs-12 col-md-8">
+		<div class="content highlighted col-xs-12 col-md-8">
+			<h2>{$ 'membership_call_to_action_title' | trans $}</h2>
+
+			<p>{$ 'membership_call_to_action_line_1' | trans $}</p>
+			<p>{$ 'membership_call_to_action_line_2' | trans $}</p>
+
+			<div class="call-to-action-box">
+				{% include featureToggle.callToActionTemplate %}
+			</div>
+
 			{% include 'partials/membership/benefit_list.html.twig' with {
 				'showActiveMembershipBenefits': true
 			} only %}
-		</div>
-		<div class="action-block col-xs-12 col-sm-offset-0 col-sm-12 col-md-8">
-			<a id="membership-advantages-button" href="{$ basepath $}/apply-for-membership?donationId={$ donation.id $}&amp;donationAccessToken={$ donation.accessToken $}&amp;type=sustaining" class="btn btn-introduction bk-blue text-center text-uppercase">
+
+			<a href="{$ basepath $}/apply-for-membership?donationId={$ donation.id $}&amp;donationAccessToken={$ donation.accessToken $}&amp;type=sustaining"
```

Comment:
`/apply-for-membership?donationId={$ donation.id $}&amp;donationAccessToken={$ donation.accessToken $}&amp;type=sustaining`

The two cases of `&amp;` in the URL look suspicious to me, I think this should be `&`.